packages/components/src/components/Input/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { uniqueId } from 'lodash-es';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
+};
+
+export const InputComponent = styled.input(
+  css({
```

Comment:
does input need a font size + font family explicitly?

Reply:
Font family will be inter for all I thin, I will add the font-size

Reply:
Do we add this in the globals?

```
body, input, select, button: {
  font-family: Inter
}
```